rocks.noise.QSpeakers.yaml

```diff
@@ -0,0 +1,21 @@
+app-id: rocks.noise.QSpeakers
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: qspeakers
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  - --device=dri
+rename-desktop-file: qspeakers.desktop
+copy-icon: qspeakers
```

Comment:
it's no like flatpak-builder is silent

```
(flatpak-builder:2): Json-WARNING **: 18:24:00.428: Failed to deserialize "copy-icon" property of type "gboolean" for an object of type "BuilderManifest"
```

I guess you didn't try your changes nor checked the documentation.

Reply:
Indeed, sorry for that. I'll do now.

Reply:
Reading the doc, i see that the convetions  is now to not Uppercase the application name part of the app-id.
Fixing that, plus the rename-icon / copy-icon issue.